charts/postgresql/templates/_helpers.tpl

```diff
@@ -303,31 +303,22 @@ Return  the proper Storage Class
 {{- define "postgresql.storageClass" -}}
 {{/*
 Helm 2.11 supports the assignment of a value to a variable defined in a different scope,
-but Helm 2.9 and 2.10 does not support it, so we need to implement this if-else logic.
+but Helm 2.9 and 2.10 do not support it, so we need to implement this if-else logic.
+This version prioritizes component-specific persistence.storageClass over global.storageClass.
```

Comment:
We don't support helm 2 at all. Can we remove these if-else blocks?

Reply:
we can remove this altogether

Reply:
done